### ISSUE TITLE
Bootstrap with filter

### ISSUE THREAD

**Issue by pollosp** [zendesk/maxwell#1037]
I'm trying to bootstrap using the following command line:

```
/usr/bin/docker run --rm --name maxwell zendesk/maxwell:v1.16.1 sh -c 'cp /app/kinesis-producer-library.properties.example /app/kinesis-producer-library.properties && echo "Region=${maxwell_region}" >> /app/kinesis-producer-library.properties && bin/maxwell --user=${mysql_username} --password="${mysql_password}" --host=${mysql_host} --producer=kinesis --metrics_type=http --http_port=80 --kinesis_stream=${kinesis_stream} "--filter="exclude: *.*, include: xxx_core.*""'
```

```
./maxwell-bootstrap -user xxx --password XXXX --log_level DEBUG --host xxxx --database xxx_core --table wwwwwwwww  --client_id maxwell
```

And bootstrap never starts , without using the filter `--filter="exclude: *.*, include: xxx_core.*""` it works perfectly

From `select * from maxwell.bootstrap;` I get 
```
+----+-------------------+-------------------+--------------+-------------+---------------+------------+------------+---------------------+--------------+----------------------------+-----------------+-----------+
| id | database_name     | table_name        | where_clause | is_complete | inserted_rows | total_rows | created_at | started_at          | completed_at | binlog_file                | binlog_position | client_id |
+----+-------------------+-------------------+--------------+-------------+---------------+------------+------------+---------------------+--------------+----------------------------+-----------------+-----------+
|  1 | xxx_core          | wwwwwwwww         | NULL         |           0 |             0 |          0 | NULL       | NULL                | NULL         | NULL                       |               0 | maxwell   |
+----+-------------------+-------------------+--------------+-------------+---------------+------------+------------+---------------------+--------------+----------------------------+-----------------+-----------+
```

I'm missing something here? thanks :)

**Comment by adamgeorgeson** [comment]
👍 I've been seeing this too. Good spot re: the filter.

**Comment by osheroff** [comment]
fixed in 1.17.1.  the sucky thing here was that we had tests around all this stuff but i broke travis-notifications awhile back.
